lib/pghero/methods/citus.rb

```diff
@@ -0,0 +1,29 @@
+module PgHero
+  module Methods
+    module Citus
+      def citus_version
+        @citus_version ||= select_one("SHOW citus.version")
+      end
+
+      def citus_available?
+        select_one("SELECT COUNT(*) AS count FROM pg_available_extensions WHERE name = 'citus'") > 0
+      end
+
+      # only cache if true
```

Comment:
It would be better if you write full sentences for comments.

Reply:
Ok.